pyuvdata/uvbeam.py

```diff
@@ -150,7 +150,9 @@ def __init__(self):
                 'Peak normalized means that for each frequency the data_array'
                 'is separately normalized such that the peak is 1 (so the beam '
                 'is dimensionless) and all frequency dependence is moved to the '
```

Comment:
Ya, I think you should just remove the end of that sentence:

 "Peak normalized means that for each frequency the data_array is separately normalized such that the peak is 1 (so the beam is dimensionless). If the beam_type is "efield", then peak normalized means that the absolute value of the peak is 1. Solid angle normalized means the peak normalized beam is divided by the integral of the beam over the sphere, so the beam has dimensions of 1/steradian."